src/components/common/Select/Select.tsx

```diff
@@ -1,2 +1,58 @@
-import { Select } from '@terraware/web-components';
+import { Select, SelectT } from '@terraware/web-components';
 export default Select;
+
+export type SelectItem = {
+  label: string;
+  value: string;
+};
+
+export interface SelectValueProps {
+  onChange: (newValue: string) => void;
+  label?: string;
+  disabled?: boolean;
+  id?: string;
+  className?: string;
+  helperText?: string | string[];
+  placeholder?: string;
+  errorText?: string | string[];
+  warningText?: string | string[];
+  selectedValue?: string;
+  readonly?: boolean;
+  options?: SelectItem[];
+  fullWidth?: boolean;
+  hideArrow?: boolean;
+  onBlur?: () => void;
+  onFocus?: () => void;
+  fixedMenu?: boolean;
+}
+
+/**
+ * This is a simple component that takes in a tuple { label, value }
+ * for list of options.
+ * The label is used for display, the value is passed back in onChange, and used to set selectedValue.
+ * This is a very common use-case for a Select component, extracting it out
+ * here. Not a candidate for web-components at this point.
+ *
+ * Example:
+ * <SelectValue
+ *   options=[{ label: 'label', value: 'value'}, { label: 'label1': value: 'value1'} ]
+ *   onChange={(value: string) => setSomeValue(value)}
+ *   selectedValue={'value1'}
+ * />
+ */
+export function SelectValue(props: SelectValueProps): JSX.Element {
```

Comment:
we could copy it to web-components as the new Dropdown.. or leave it here

Reply:
I could have use this select on my quantity PR, and also withdrawal.

Reply:
ha ha yes, i added a comment in your PR. I noticed we have this use-case a lot.
i can merge this in sooner.. i will move it to Dropdown/web-components later and make relevant updates..